mockery/generator.go

```diff
@@ -277,6 +278,15 @@ func (g *Generator) genList(list *ast.FieldList, addNames bool) ([]string, []str
 	return names, types, params
 }
 
+func (g *Generator) paramsAsArgNames(params []string) []string {
+	args := []string{}
+	r := regexp.MustCompile("(.+?) (\\.{3})?.+")
```

Comment:
Mockery normalizes the params up above directly from the AST, so this
should be ok, but I can test it to make sure.

On Friday, October 16, 2015, Fabio Kung <notifications@github.com> wrote:

> In mockery/generator.go
> <https://github.com/ryanbrainard/mockery/pull/3#discussion_r42300369>:
>
> > @@ -277,6 +278,15 @@ func (g *Generator) genList(list *ast.FieldList, addNames bool) ([]string, []str
> >  	return names, types, params
> >  }
> >
> > +func (g *Generator) paramsAsArgNames(params []string) []string {
> > +	args := []string{}
> > +	r := regexp.MustCompile("(.+?) (\\.{3})?.+")
>
> the space in between the param name and the ellipsis (...) is optional in
> Go
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ryanbrainard/mockery/pull/3/files#r42300369>.
>